docs/symfony-integration.md

```diff
@@ -121,9 +121,11 @@ use OpenTelemetry\API\Trace as API;
 use Symfony\Component\Dotenv\Dotenv;
 use Symfony\Component\ErrorHandler\Debug;
 use Symfony\Component\HttpFoundation\Request;
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\HttpFactory;
 ```
 
-Next, we create a sample recording trace using the [AlwaysOnSampler](https://github.com/open-telemetry/opentelemetry-php/blob/main/sdk/Trace/Sampler/AlwaysOnSampler.php) class, just before the Kernel instance is created like below:
+Next, we create a sample recording trace using the [AlwaysOnSampler](https://github.com/open-telemetry/opentelemetry-php/blob/main/src/SDK/Trace/Sampler/AlwaysOnSampler.php) class, just before the Kernel instance is created like below:
```

Comment:
Paragraph should be removed - sampling is handled in span builder.

Reply:
Updated